src/client/components/Navigation/Navigation.tsx

```diff
@@ -15,9 +15,12 @@ const Navigation: React.FC = () => {
   const countryIso = useCountryIso()
   const dispatch = useAppDispatch()
   const { print } = useIsPrint()
+  const isInGeoPage = useIsGeoPage()
+
   useEffect(() => {
-    if (print) dispatch(NavigationActions.updateNavigationVisible(false))
-  }, [print, dispatch])
+    if (print || isInGeoPage) dispatch(NavigationActions.updateNavigationVisible(false))
```

Comment:
If this is not rendered, for some reason the map doesn’t render

This doesn’t cause an animation though, so I would leave as is if this is ok, otherwise I can also investigate why the map doesn’t render when this is hidden


Which is preferred?

Reply:
Great, the issue was only that there was no min-width.

Reply:
<img width="1444" alt="Screenshot 2023-08-02 at 15 35 36" src="https://github.com/openforis/fra-platform/assets/5508251/c5f58868-3430-4433-959b-4d5b69d5d663">